library/src/helpers/server.ts

```diff
@@ -59,13 +59,12 @@ export class ServerHelpers {
       } else {
         securityProtocol = 'PLAINTEXT';
       }
+    } else if (securitySchema) {
+      securityProtocol = 'SASL_SSL';
     } else {
-      if (securitySchema) {
```

Comment:
don't you think that it would actually be better to fix it differently, also improving the nested if above your fixed code

```
securityProtocol = (protocol === 'kafka') 
  ? (securitySchema ? 'SASL_PLAINTEXT' : 'PLAINTEXT')
  : (securitySchema ? 'SASL_SSL' : 'SSL');
```

I think ternary condition like this would be more readable, thoughts?

Reply:
yes it would be less messy and looking more clean